docs/topics/db/models.txt

```diff
@@ -525,6 +525,7 @@ the intermediate model:
 
     >>> ringo = Person.objects.create(name="Ringo Starr")
     >>> paul = Person.objects.create(name="Paul McCartney")
+    >>> john = Person.objects.create(name="John Wick")
```

Comment:
To people who know this band, it might incorrectly suggest a deliberate choice to create a different John if we don't use the John from the Beatles.

```suggestion
    >>> john = Person.objects.create(name="John Lennon")
```

Reply:
I'd add it to the second code block when it's actually used.

Reply:
I was gunna say that it wasn't worth worrying about as most examples have bits missing? 🤔 

Eg: In the second block `george` is also undefined. It gets created in the db but there's no var declaration.